pcer/main.py

```diff
@@ -1,9 +1,11 @@
 import sys
 from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import QTimer
 from experiment_controller import ExperimentController
 
 if __name__ == '__main__':
     app = QApplication(sys.argv)
+    timer = QTimer()
```

Comment:
We might need our own timer PcerTimer, properly designed to be added to an status bar. BTW, there is a QtStatusBar that we can use